tests/integration/AllUnitTestsInplaceTest.php

```diff
@@ -6,7 +6,7 @@
 
 final class AllUnitTestsInplaceTest extends TestCase {
 	public function testAllUnitTestsInPlace() {
-		$files_arr = scandir("tests/");
+		$files_arr = scandir("tests/integration");
```

Comment:
Maybe we should we include the `test/unit` folder as well?

Reply:
Sounds good. We can use ``phpunit.xml`` config file to filter them. I will do that

Reply:
I'm unsure if I understand the goal of this test. Could you elaborate a bit about it?

Reply:
If a test file does not have the `Test.php` ending, it will not be run. It can easily happen by accident that a test is created and the `Test` ending is not included, leading to the unit test never being executed. This test will fail if a file is added that does not have this ending (unless it is added to the whitelist).

Reply:
Thanks for explaining!
It seems this test tests the integration tests directory itself; I don't think it's necessary to add a unit test to test its own directory for the moment.